storage/gcloud/aio/storage/storage.py

```diff
@@ -148,6 +148,7 @@ def get_bucket(self, bucket_name: str) -> Bucket:
 
     async def copy(self, bucket: str, object_name: str,
                    destination_bucket: str, *, new_name: Optional[str] = None,
+                   data: Optional[str] = None,
```

Comment:
What do you think about calling this `metadata` instead? `data` is fairly broad and it looks like [this endpoint only supports object metadata in the body](https://cloud.google.com/storage/docs/json_api/v1/objects/rewrite) anyway.

Reply:
Hey @TheKevJames I agree with you that object metadata is being should be set with this parameter. Therefore, I changed the naming to `metadata`. The original intention was to expose the `data` parameter from the post request, see here: https://github.com/langered/gcloud-aio/blob/908603ba96812d7fe88e7416d66a55ddb77a492d/storage/gcloud/aio/storage/storage.py#L198-L199

Also, I see the risk of confusion, because you still have to pass the metadata as a attribute inside the metadata object like: `{metadata: {...}}` as described here: https://cloud.google.com/storage/docs/json_api/v1/objects#resource